llvm/test/CodeGen/X86/tailcc-ssp.ll

```diff
@@ -5,9 +5,20 @@ declare void @h(ptr, i64, ptr)
 
```

Comment:
Can we use update_llc_test_checks.py for this test?

Reply:
done